lrng_es_mgr.c

```diff
@@ -104,7 +104,6 @@ bool lrng_enforce_panic_on_permanent_health_failure(void)
 
 bool lrng_ntg1_2022_compliant(void)
 {
-	/* Implies use of /dev/random w/ O_SYNC / getrandom w/ GRND_RANDOM */
```

Comment:
I deleted this comment because the way how I understand is that this applies only on version 2.0 (2011) and it is not necessary for 3.0. And this check is really just for checking 2024 compliance. Please let me know if I missunderstood it and will put it back.